spec/jobs/long_running_job.rb

```diff
@@ -0,0 +1,7 @@
+class LongRunningJob
+  include Sidekiq::Worker
+  sidekiq_options queue: :customqueue, retry: true, unique: :until_and_while_executing,
+                  run_lock_expiration: 7_200, retry_count: 10
```

Comment:
Ok got it with these names it makes for sense writing out the full thing with `default_yada`.